apps/waterfall/waterfall/src/app/dashboard/home/home.component.ts

```diff
@@ -1,21 +1,49 @@
 // Angular
-import { Component, ChangeDetectionStrategy } from '@angular/core';
+import { switchMap, tap } from 'rxjs';
+import { Router } from '@angular/router';
+import { Component, ChangeDetectionStrategy, Inject } from '@angular/core';
 
 // Blockframes
+import { APP } from '@blockframes/utils/routes/utils';
+import { sorts } from '@blockframes/ui/list/table/sorts';
+import { filters } from '@blockframes/ui/list/table/filters';
+import { App, Movie, hasAppStatus } from '@blockframes/model';
+import { OrganizationService } from '@blockframes/organization/service';
+import { MovieService, fromOrgAndAccessible } from '@blockframes/movie/service';
 import { DynamicTitleService } from '@blockframes/utils/dynamic-title/dynamic-title.service';
 
+
 @Component({
   selector: 'dashboard-home',
   templateUrl: './home.component.html',
   styleUrls: ['./home.component.scss'],
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class HomeComponent {
- 
+
+  public sorts = sorts;
+  public filters = filters;
+
+  public titles$ = this.orgService.currentOrg$.pipe(
+    switchMap(({ id }) => this.movieService.valueChanges(fromOrgAndAccessible(id, this.app))),
+    tap(titles => {
+      titles.filter(hasAppStatus(this.app, ['accepted', 'submitted'])).length
+        ? this.dynTitle.setPageTitle('Dashboard')
+        : this.dynTitle.setPageTitle('Dashboard', 'Empty');
```

Comment:
I think this can be updated since movie status will not be taken into account on this app. 
```suggestion
      titles.length
        ? this.dynTitle.setPageTitle('Dashboard')
        : this.dynTitle.setPageTitle('Dashboard', 'Empty');
```